numba_cuda/numba/cuda/tests/cudapy/test_print.py

```diff
@@ -105,6 +118,12 @@ def test_string(self):
         expected = ['%d hop! 999' % i for i in range(3)]
         self.assertEqual(sorted(lines), expected)
 
+    def test_dim3(self):
+        output, _ = self.run_code(printdim3_usecase)
+        lines = [line.strip() for line in output.splitlines(True)]
+        expected = [str((k, j, i)) for i in range(2) for j in range(2) for k in range(2)]
+        self.assertEqual(sorted(lines), sorted(expected))
```

Comment:
Line 124 is a bit long for flake8 - this can also be simplified with `ndindex` (note you will need to add `import numpy as np` to the imports in this file as well):

```suggestion
        expected = [str((i)) for i in np.ndindex(2, 2, 2)]
        self.assertEqual(sorted(lines), expected)
```

As a small comment - I think using three nested for loops in a list comprehension could in general be seen as a little unidiomatic, seeming a bit complex for a one-liner.